membership_extension/views/res_partner_view.xml

```
@@ -40,20 +40,20 @@
                 expr="//field[@name='member_lines']/tree//field[@name='date']"
                 position="after"
             >
-                <field name="date_from" />
-                <field name="date_to" />
+                <field name="date_from" optional="hide" />
```

This should be shown by default IMO, and isn't this the most important info in the line?

---

Ops, sorry. 
I swapped show/hide. :sweat_smile:

---

Fixed.

---

But as said, do you think that 2 fields should be optional? Without them, the line loses its meaning.

---

membership_extension adds many features, and not everyone needs all of them.
Besides, adding 4 fields in that view make it less readable as fields width get smaller.
My idea is to show only the most important ones (date_from and date_to) in the view (missing in the original module).

What do you think?

---

Yeah, but as optional?

---

They are shown by default but users could choose to hide them, more freedom.

If you think it's not worth it I can remove those options for date_from/date_to.